scania-digital-design-system/sdds
# AB#848 Improvement/form fields

- Added native select option classes to style with SDDS dropdown
- Re arrange scss dropdown files structure in components
- Added more CSS Vars in dropdown theme for more flexibility
- Added more examples of using dropdown and textfield in React and Angular
- Added routing to both demo 

Fix [AB#848](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/848)

## How to test:

1. Open demo in react and angular
2. Link to this branch with npm link
3. Can also check in Storybook > Components > Dropdown > Native dropdown
